model/src/main/kotlin/config/LicenseFilePatterns.kt

```diff
@@ -61,6 +66,9 @@ data class LicenseFilePatterns(
                 "unlicence",
                 "unlicense"
             ),
+            noticeFilenames = setOf(
+                "notice*"
```

Comment:
Have you seen examples, which need the `*`?

Reply:
Yes, `NOTICE.txt` for example, as also mentioned at https://infra.apache.org/licensing-howto.html#source-tree-location.

Reply:
Maybe, using two values: `notice` and `notice.*` could make sense?

Reply:
I was considering this, but then realized that we also use "license*" etc., so I felt it's not needed to limit to extensions here (unless we do the same for license files as well).